screens/Browsing.tsx

```diff
@@ -90,7 +91,7 @@ export function Browsing() {
           if (publisherCode) {
             navigation.dispatch(
               StackActions.replace("Story", {
-                id: "narou__" + publisherCode,
+                storyId: "narou__" + publisherCode,
```

Comment:
This property name is changed in https://github.com/compac-reader/compac-reader/pull/29